[Bug] Fix text alignment and enable href in Button (#881)
mckinsey/vizro: vizro-core/docs/pages/user-guides/card-button.md

> ### Create a navigation card

Author: For review for @stichbury

This section describes how to use the [`Card`][vizro.models.Card] component to create a navigation card,
enabling users to navigate to another page by clicking on the card area.

> For a button-style navigation component, see the [guide on creating a link button](#create-a-link-button).

Reviewer: ```suggestion
For a button-style link navigation component, see the [separate guide on creating a link button](#create-a-link-button).
```

> To configure the navigation panel on the left hand side of the screen, refer to the [guide on navigation](navigation.md).

Reviewer: ```suggestion
To configure the navigation panel on the left hand side of the screen, refer to the [separate guide on navigation](navigation.md).
```

such as form submissions, navigation links, and other action triggers.

To add a [`Button`][vizro.models.Button], insert it into the `components` argument of the
[`Page`][vizro.models.Page].

### Customize button text

You can configure the `text` argument to alter the display text of the [`Button`][vizro.models.Button].

!!! example "Customize text"

    === "app.py"
        ```{.python pycafe-link}
        import vizro.models as vm
        from vizro import Vizro

        page = vm.Page(
            title="Button with text",
            components=[vm.Button(text="I'm a button!")],
        )

        dashboard = vm.Dashboard(pages=[page])
        Vizro().build(dashboard).run()
        ```
    === "app.yaml"
        ```yaml
        # Still requires a .py to add data to the data manager and parse YAML configuration
        # See from_yaml example
        pages:
          - components:
            - type: button
              text: I'm a button!
            title: Button with text
        ```
    === "Result"
        [![ButtonText]][ButtonText]

    [ButtonText]: ../../assets/user_guides/components/button_text.png

### Create a link button

> To navigate to different pages using a button with an anchor tag, provide an absolute or relative URL to the

Reviewer: ```suggestion
To navigate to different pages using a button with an anchor tag, assign an absolute or relative URL to the
```

`Button.href`.

```python
vm.Button(text="Leave us a star! ⭐", href="https://github.com/mckinsey/vizro")
```

### Attach an action

You can use the [`Button`][vizro.models.Button] to trigger predefined action functions, such as exporting data.
To explore the available options for [`Actions`][vizro.models.Action], refer to our [API reference][vizro.actions].

> Use the `Button.actions` argument to specify which action function should be executed when the button is clicked.

Reviewer: ```suggestion
Use the `Button.actions` argument to specify which action function executes when the button is clicked.
```

> In the example below, we demonstrate how to configure a button to export the filtered data of a target chart using the

Reviewer: ```suggestion
The example below demonstrates how to configure a button to export the filtered data of a target chart using the
```
